functional_tests/steps/home_page.py

```diff
@@ -1,9 +1,9 @@
-from behave import then, when  # pylint: disable=E0611
+from behave import then, when  # pylint: disable=no-name-in-module
```

Comment:
Have you tried this? https://pypi.org/project/pylint-behave/

Reply:
I'll look into it. It looks like the issue is already fixed in the Behave source, but the next release that'll include it has been in development for a few years already, so no indication when it'll be available. Hopefully the plugin will stop us having to ignore everywhere 👍

Reply:
Tried the plugin, it does deal with the Behave false warnings, but it also caused new false warnings from a different, unrelated file, so I won't introduce it for now, perhaps can be revisited in a future iteration.